lib/host.py

```diff
@@ -62,6 +62,12 @@ def stringify(key, value):
                 simple_output=simple_output
             )
 
+        if split is not None:
+            result_as_list = []
+            if len(result) > 0:
+                result_as_list = result.split(split)
+            return result_as_list
```

Comment:
I'm not sure if this is the right place for this code. This helper can be generic.
I think this new `split` param can be removed,  and this block could be moved in a new function executed after the `xe` call.